include/gul14/num_util.h

```diff
@@ -71,7 +71,7 @@ constexpr auto abs(ValueT n) noexcept -> std::enable_if_t<not std::is_unsigned<V
  * orders equals digits does not hold so strict anymore.
  *
  * Remember that any nonzero number has infinite different significant digits compared
- * with 0.00000000. So if either a or b is 0.0 the result must be false.
+ * with 0.00000000. So if either a or b (not both) is 0.0 the result must be false.
```

Comment:
Maybe we should also spell out explicitly that the function returns true if a == b == 0.

Reply:
Done